tensorflow/core/kernels/cudnn_rnn_ops.cc

```diff
@@ -826,14 +826,33 @@ Status DoForward(OpKernelContext* context, const RnnDescriptor& rnn_desc,
   }
 
   Stream* stream = context->op_device_context()->stream();
+
+  Tensor seq_lengths_tensor;
+  DeviceMemory<int> seq_lengths_ptr;
+  if (sequence_lengths != nullptr) {
+	  auto seq_lengths_vec = sequence_lengths->template flat<int>();
```

Comment:
Can we use the `absl::Span<const int> seq_lengths` created above?

Reply:
Yes, this is a good point. I just removed them in the new commit. PTAL.